tests/unit/info.tcl

```diff
@@ -6,9 +6,64 @@ proc errorstat {cmd} {
     return [errorrstat $cmd r]
 }
 
+proc latency_percentiles_usec {cmd} {
+    return [latencyrstat_percentiles $cmd r]
+}
+
+proc latency_hist_usec {cmd} {
+    return [latencyrstat_hist $cmd r]
+}
+
 start_server {tags {"info" "external:skip"}} {
     start_server {} {
 
+        test {latencystats: disable/enable} {
```

Comment:
maybe we can somehow add one test that actually measures latency? compare a PING and a DEBUG SLEEP

Reply:
@oranagra added "latencystats: measure latency" test, and also included the percentile check on the module's "blocked clients time tracking..." test.